src/controllers/users/signup.js

```diff
@@ -0,0 +1,41 @@
+import userModel from '../../models/index'
+import dotenv from 'dotenv';
+import bcryptHash from '../../helpers/bcryptHash';
+
+dotenv.config();
+
+const User = userModel.User;
+
+class SignClass{
+    static signup(req, res) {
+        const hashedPassword = bcryptHash.hashFunction(req.body.password);
+        const {firstname, lastname, email} = req.body;
+        const userSchema = {firstname,lastname,email,password: hashedPassword}
+
+        User.create(userSchema)
+        .then((user) => {
+            const {id, firstname, lastname, email} = user;
+            const payload = {id,firstname,lastname,email};
+            
+            const token = bcryptHash.tokenFunction(payload, process.env.SECRET_KEY)
+            res.status(201).json({
```

Comment:
I suggest we use the `return` key word to end the request processing cycle at this point.